### DISCUSSION TITLE
话说，最近点击视频变成在当前页面打开，而不是在新标签页？

### DISCUSSION THREAD

**Discussion by naive12138** [the1812/Bilibili-Evolved#1302] [提问 / Q&A] [answered]
最近点击视频变成在当前页面打开，而不是在新标签页？这个问题是插件引起的还是chrome本身的问题又或是B站的问题。
插件版本号：v1.11.5 内容包: 27accbb
chrome版本 87.0.4280.88（正式版本）

**Comment by Coulomb-G** [comment, marked as answer]
你说的是在哪个页面点击视频?我这边首页和动态点击视频都是打开新页,关闭脚本再看看?

**Reply by naive12138** [reply]
谢谢，那应该是我的chrome浏览的问题。

**Reply by naive12138** [reply]
搞懂了，原来是我的另一个插件引起的，插件自动更新了一个我也不知道是啥憨憨功能导致的这个问题。再次感谢回复

**Reply by happytbs** [reply]
请问是哪个插件引起的，我也遇到了同样的问题
